web-console/knikubevirt/knikubevirt.md

```diff
@@ -0,0 +1,31 @@
+## KNI and KubeVirt Designs
+Access design documentation specific to the KNI and KubeVirt features in OpenShift.
+
+### OpenShift 4.2
+- Cluster Dashboard
+- [VM List](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/vm-list/vm-list)
+- [VM Templates](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/vm-templates/vm-templates)
+- [VM Details](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/vm-details/vm-details)
+
+### Future Designs
+
+#### KNI (Kubernetes Native Infrastructure/OpenShift on Bare Metal)
+- Storage Dashboard
+- Add Bare Metal Host
+- Remove Bare Metal Host
+- Update Bare Metal Cluster
+
+#### KubeVirt (Virtualization in OpenShift)
+- [Clone VM](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/clone-vm/clone-vm)
+- Create VM
```

Comment:
This one is tricky...we will have Adding a VM in 4.2 but it won't be the PF4 design that Matt is pushing...should it still be in 4.2?

Reply:
ya I wondered that too. Is it worth for the sake of clarity to have a "4.2" design in here as well for the create flow? It's a reasonable amount of effort but could be worthwhile. WDYT?

Reply:
How about just a large banner stating that for 4.2 it will be PF3? I'm all for reducing the amount of work on it in any way we can :)

Reply:
+1 yes, if that's enough to avoid confusion I am more than happy with that :)

Reply:
I know the functionality will exist in 4.2 but this design is really for 4.3 so I would rather not add this as a 4.2 design since it looks quite different from the PF3 implementation.

Reply:
@beanh66 - Makes sense to leave this out then. I think when @matthewcarleton and I chatted yesterday late afternoon we talked in a circle and ended up here too :)